index.bs

```diff
@@ -1662,6 +1719,13 @@ To <dfn>obtain and deliver a debug report on source registration</dfn> given a
 
 To <dfn>process an attribution source</dfn> given an [=attribution source=] |source|:
 
+1. Let |destinationRateLimitResult| be the result of running [=check if an attribution source exceeds the time-based destination limit=] with |source|.
+1. If |destinationRateLimitResult| is "<code>[=destination rate-limit result/hit reporting limit=]</code>":
+    1. Run [=obtain and deliver a debug report on source registration=] with "<code>[=source debug data type/source-destination-rate-limit=]</code>" and |source|.
```

Comment:
The new debug report type should be handled in "check if debug reporting is allowed" and "obtain and deliver a debug report on source registration" algorithms.

Reply:
Done